shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -13,10 +13,16 @@
 
 static UIKeyboardType ToUIKeyboardType(NSDictionary* type) {
   NSString* inputType = type[@"name"];
-  if ([inputType isEqualToString:@"TextInputType.text"])
+  if ([inputType isEqualToString:@"TextInputType.address"])
```

Comment:
This type of keyboard does not trigger autofill. Developers would have to use `TextInputType.name` instead to trigger street autofill which seems really counterintuitive. But `TextInputType.name` (`UIKeyboardTypeNamePhonePad`) keyboard doesn't have punctuation.